cypress/support/elements/common/PrimaryWorkspace.js

```diff
@@ -36,28 +36,40 @@ class RightNav{
         return this.testDocumentWithTitle(tab, subTab, title, 'not.contain');
     }
 
-    getRightNavTabs(){
+    getPrimaryWorkspaceTabs(){
         return cy.get('.right-nav .tabs');
     }
 
-    closeRightNavTabs(){
+    closePrimaryWorkspaceTabs(){
         cy.get('.close-button').click();
         cy.wait(1000);
     }
 
-    getRightNavTab(tab){
-        return cy.get('.nav-tab.tab-'+tab);
+    getPrimaryWorkspaceTab(tab){
+        return cy.get('.top-tab.tab-'+tab);
```

Comment:
All tabs are now in the resource panel and should probably be consolidated there instead of here.

Reply:
Done.